docs/pages/guides/transactions.mdx

```diff
@@ -18,6 +18,36 @@ buttons: [
 ];
 ```
 
+In your `/txdata` route you should handle the request and return transaction data that conforms to the [Transaction schema](/schemas/transaction).
```

Comment:
missing fully qualified URL to transaction schema. Could you fix it @stephancill ? It fails vercel build.

Reply:
good catch, thanks. fixed